network-api/networkapi/donate/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from wagtail.admin.panels import FieldPanel
+from wagtail.fields import RichTextField
+from wagtail.models import Page
+from wagtail_localize.fields import SynchronizedField, TranslatableField
+
+from networkapi.wagtailpages.models import FoundationMetadataPageMixin
+
+
+class DonationPage(FoundationMetadataPageMixin, Page):
```

Comment:
The `DonationPage` abstract model is empty for now, is going to be inherited by `DonateLandingPage` and `DonateCampaignPage`.

That way both pages inherit the FoundationMetadataPageMixin, and fields that we add at a later time.

Reply:
I'm not sure what the naming convention is but if it's an abstract model, would it be possible if we rename it to something else? Looking at our codebase it seems like "mixin" or "base" were used for other abstract models.

Reply:
Thanks for the suggestion Mavis! 

This sounds like a great idea to me. I have updated the `DonationPage` model to now be named `BaseDonationPage`.